src/bout++.cxx

```diff
@@ -83,6 +83,9 @@ void bout_signal_handler(int sig);  // Handles signals
 
 #include <output.hxx>
 
+// We initialize the global variables here, so allow their use
+using namespace bout::globals;
```

Comment:
I'm not sure I understand the comment? We're using the variables a lot here, so the `using namespace` is for convenience?

Reply:
It was for convenience, but on second thoughts I don't like the `using namespace` here. Will get rid of it.